Added info about how to disable auto resize while capturing bitmap (#1152)
100mslive/100ms-docs: docs/android/v2/features/render-video.mdx

Until you call `removeTrack()` the video frame is being streamed over the device's data.

### Capture Video Frame

> You can get a bitmap object of the frame getting rendered on the `HMSVideoView`

Reviewer: ```suggestion
You can get a bitmap object of the frame getting rendered on the `HMSVideoView`.

```

> In case you are calling this API on a `videoView` which is rendering localVideoTrack, it will return a bitmap whose resolution would be equal to the resolution at which this peer is supposed to publish

Reviewer: ```suggestion
In case you are calling this API on a `videoView` which is rendering `localVideoTrack`, it will return a bitmap whose resolution would be equal to the resolution at which this peer is supposed to publish.

```

Until you call `removeTrack()` the video frame is being streamed over the device's data.

### Capture Video Frame

> You can get a bitmap object of the frame getting rendered on the HMSVideoView

Reviewer: ```suggestion
You can get a bitmap object of the frame getting rendered on the `HMSVideoView`

```

> Incase you are calling this API on a `videoView` which is rendering localVideoTrack, it will return a bitmap whose resolution would be equal to the resolution at which this peer is supposed to publish

Reviewer: ```suggestion
In case you are calling this API on a `videoView` which is rendering localVideoTrack, it will return a bitmap whose resolution would be equal to the resolution at which this peer is supposed to publish

```

> Incase you are calling this API on a `videoView` which is rendering localVideoTrack, it will return a bitmap whose resolution would be equal to the resolution at which this peer is supposed to publish

Reviewer: shouldnt localVideoTrack be written as `localVideoTrack` ?
